src/openforms/payments/base.py

```diff
@@ -2,13 +2,13 @@
 from typing import TYPE_CHECKING, Mapping
 
 from django.http import HttpRequest, HttpResponse
-from django.utils.translation import gettext_lazy as _
 
 from rest_framework import serializers
 from rest_framework.reverse import reverse
 
 from openforms.utils.mixins import JsonSchemaSerializerMixin
 
+from ..plugins.plugin import AbstractPlugin
```

Comment:
please use absolute imports when importing outside of the django app/openforms module